### PR TITLE
doc:  update tco

### PR DESCRIPTION
**Pull request by Marchabar**
# Description
- Add sendgrid costs in opex
- Add new TCO excel for 2 years, differentiating between 4 monts of development and the rest of months
- I will update the current tco status for weeks 8 and 9, on Tuesday 2nd April before the evaluation, as usual.
# Checklist:
- [X] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [X] I have checked that there is no fragile information published unintentionally.

# Screenshot:
image
image

### CODE REVIEW

File: docs/Coste total de la propiedad/TCO.mdx [hunk 1]
```diff
@@ -216,9 +230,9 @@ Teniendo en cuenta la estimación de costes de desarrollo del proyecto, y la est
 </iframe>
 
 
-El coste total de la propiedad anual es de **74.5K**.
+El coste total de la propiedad anual es de **91.2K**.
```
Comment (alvgonfri) [inline docs/Coste total de la propiedad/TCO.mdx:233]: You should change the word "annual" to say it's now a 2-year TCO.

**Review by alvgonfri — CHANGES_REQUESTED** [review]
It looks correct overall, but there is a small change to do, which you can see below.

Additional note: you said in this PR description that you will be updating the current TCO status for weeks 8 and 9 on Tuesday 2nd April, but if I'm not wrong week 9 starts on 2nd April.

**Comment by Marchabar** [comment]
> It looks correct overall, but there is a small change to do, which you can see below.
> 
> Additional note: you said in this PR description that you will be updating the current TCO status for weeks 8 and 9 on Tuesday 2nd April, but if I'm not wrong week 9 starts on 2nd April.

I have updated what you suggested. About the weeks: week 8 is on 19th-25th of March and week 9 is on 26th-1st of April. So, on tuesday 2nd I will update both weeks closing then sprint 2.

**Review by alvgonfri — APPROVED** [review]
Everything is correct now!

**Pull request merged**